src/hotspot/share/runtime/arguments.cpp

```diff
@@ -3785,17 +3785,17 @@ static void apply_debugger_ergo() {
 
 static const char* const hotspotrc = ".hotspotrc";
 
-class Arguments::PreprocessedImpl final : public CHeapObj<mtArguments> {
+class PreprocessedArguments final : public CHeapObj<mtArguments> {
  public:
-  PreprocessedImpl() : initial_vm_options_args(""),
-                       initial_java_tool_options_args("env_var='JAVA_TOOL_OPTIONS'"),
-                       initial_java_options_args("env_var='_JAVA_OPTIONS'"),
-                       mod_cmd_args("cmd_line_args"),
-                       mod_vm_options_args("vm_options_args"),
-                       mod_java_tool_options_args("env_var='JAVA_TOOL_OPTIONS'"),
-                       mod_java_options_args("env_var='_JAVA_OPTIONS'") {}
+  PreprocessedArguments() : initial_vm_options_args(""),
+                            initial_java_tool_options_args("env_var='JAVA_TOOL_OPTIONS'"),
+                            initial_java_options_args("env_var='_JAVA_OPTIONS'"),
+                            mod_cmd_args("cmd_line_args"),
+                            mod_vm_options_args("vm_options_args"),
+                            mod_java_tool_options_args("env_var='JAVA_TOOL_OPTIONS'"),
+                            mod_java_options_args("env_var='_JAVA_OPTIONS'") {}
```

Comment:
Indentation is off - args should align on the `"` of the first arg.

Reply:
How do you mean? Like do not align the variables, but align the arguments, or both? Do you have an example?

Reply:
Never mind - seems to be a glitch in the way the diff is presented. The actual file content appears fine.